02LessonTwo/homework/css/style.css

```diff
@@ -1,31 +1,48 @@
 body{
-	font-size: 20px;
-	background-color: gray;
+	font-size: 22px; font-style: Helvetica;
 }
 header{
```

Comment:
Make sure to add spaces between the selector and the brackets when creating your css selector/properties etc.